src/main/kotlin/no/nav/dagpenger/inntekt/InntektApi.kt

```diff
@@ -47,6 +49,9 @@ fun Application.inntektApi(env: Environment, inntektskomponentHttpClient: Inntek
     install(DefaultHeaders)
 
     install(StatusPages) {
+        exception<BadRequestException> { cause ->
```

Comment:
Hvor ikke bare respondere  `call.respond(HttpStatusCode.BadRequest)`  (see under) direkte her?

Reply:
btw, sjekk https://github.com/navikt/dp-inntekt-api/pull/5 for å returnere "http problem"